examples/components/NER/traininsilo/run.py

```diff
@@ -180,13 +180,13 @@ def __init__(
         logger.info(f"Trainable parameters: {trainable_params}")
 
         self.model_.train()
+        self.model_.to(self.device_)
         if self._distributed:
             self.model_ = DDP(
```

Comment:
You're moving the call to .to(device) much earlier than before, is there a reason for it?

In particular, when you instanciate DDP(), I don't know how that works with .to(), is it being taken care of internally in DDP()?

Reply:
forget about this comment, no pb here